build/js/Game.js

```diff
@@ -93,12 +93,17 @@ Game.prototype.assignRandomColors = function() {
   var cardbacks = document.querySelectorAll('.card-back');
   var color;
 
-  // Add 6 random hexcodes to the randomColors array.
-  for(var i = 0; i < 6; i++) { 
-    color = this.generateRandomHexcode();
+  // array of colors to choose from
+  var colorOptions = ['#333', '#666', '#999', '#ccc', '#000033', '#000066', '#000099', '#0000cc', '#0000ff', '#003300', '#006600', '#009900', '#00cc00', '#00ff00', '#00ffcc', '#00ccff', '#00ffff', '#330000', '#660000', '#990000', '#cc0000', '#ff0000', '#660066', '#660099', '#9900ff', '#ccff00', '#ccffcc', '#99ff00', '#ff9900', '#ffcc33'];
```

Comment:
I like most of the colors.. there still seems to be a lot of similar colors though.

Maybe if we only put in only one light and one dark combination of a certain color?

Reply:
I will make there be more variety

Reply:
Improved color contrast